scripts/test_update.sh

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/env bash
+
+set -eo pipefail
+
+SCRIPTS_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
+CONTROLLER_BOOTSTRAP_DIR="$SCRIPTS_DIR/.."
+CONTROLLER_BOOTSTRAP="$CONTROLLER_BOOTSTRAP_DIR/bin/controller-bootstrap"
+NOTICE_TPL_FILE="$CONTROLLER_BOOTSTRAP_DIR/templates/NOTICE.tpl"
+AWS_SERVICE="eks"
+CONTROLLER_NAME="$AWS_SERVICE-controller"
+OUTPUT_DIR="$CONTROLLER_BOOTSTRAP_DIR/test_output"
+CONTROLLER_DIR="$OUTPUT_DIR/$CONTROLLER_NAME"
+TEXT_TO_FIND="'this line is added by ACK update test'"
+
+if ! grep -wq -- "$TEXT_TO_FIND" "$CONTROLLER_DIR/NOTICE"; then
+    echo "Unable to find $TEXT_TO_FIND in the 'NOTICE' file of $CONTROLLER_NAME. Adding $TEXT_TO_FIND in controller-bootstrap/templates/NOTICE.tpl file ..."
+    echo "$TEXT_TO_FIND" >> $NOTICE_TPL_FILE
+    echo "Updating the project description files in the existing $CONTROLLER_NAME"
+    cd "$CONTROLLER_BOOTSTRAP_DIR"
+    make build
+    ${CONTROLLER_BOOTSTRAP} update --aws-service-alias ${AWS_SERVICE} --output-path ${CONTROLLER_DIR}
```

Comment:
isn't there a `make` target for executing update? If yes, use that instead.

Reply:
Currently in Makefile, there isn’t a specific make target for executing update. However, I can either replace the above with `make` command, which checks for an existing controller and updates the project description files, or add the specific make target for update. Thank you.